src/hotspot/share/gc/shenandoah/shenandoahUtils.hpp

```diff
@@ -44,6 +44,19 @@
 class GCTimer;
 class ShenandoahGeneration;
 
+#define SHENANDOAH_RETURN_EVENT_MESSAGE(generation_type, prefix, postfix) \
+  switch (generation_type) {                                              \
+    case GLOBAL:                                                          \
+      return prefix "" postfix;                                           \
```

Comment:
Suggest reporting GLOBAL here.

Reply:
Ok, now reporting `GLOBAL` for generational mode. This macro is now also checking the generational mode, so we drop any mention of generation in non-generational mode. This keeps the logs the same as upstream for non-generational mode.